ntbs-service/Pages/LegacyNotifications/Index.cshtml

```diff
@@ -0,0 +1,54 @@
+﻿@page "/LegacyNotifications/{LegacyNotificationId}"
+
+@using ntbs_service.Helpers
+@model ntbs_service.Pages.LegacyNotifications.Index
+
+@{
+    var classSuffix = CssClassHelpers.GetClassSuffixForNotificationStatus(Model.NotificationBanner.NotificationStatus);
+    var notificationStatusTriangleCssClass = $"notification-banner-triangle notification-banner-triangle{classSuffix}";
+}
+
+<nhs-width-container container-width="Standard">
+
+    <nhs-grid-row>
+        <nhs-grid-column grid-column-width="Full">
+            <div class="notification-banner-container">
+                <partial name="../Shared/_NotificationBanner.cshtml" model="@Model.NotificationBanner" />
+                <div class="@notificationStatusTriangleCssClass"> </div>
+            </div>
+        </nhs-grid-column>
+    </nhs-grid-row>
```

Comment:
I feel like we could, and probably should, use the `NotificationLayout` here.. We might need to extract an interface for its backing model, but it would allow us to not do all oft his repetition!

Reply:
Should I create another base class that NotificationBase and LegacyNotifications extend and use that on a _NotificationLayout?

Reply:
AFAIK (haven't checked in depth) NotificationLayout only needs a notificationBannerModel so you might be able to just get away with passing the banner model in to it wherever we use it